akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2StreamHandling.scala

```diff
@@ -229,6 +235,10 @@ private[http2] trait Http2StreamHandling { self: GraphStageLogic with StageLoggi
     override def shutdown(): IncomingStreamState = Closed
   }
   case object Closed extends IncomingStreamState {
+
+    override def handleOutgoingCreated(stream: Http2SubStream): IncomingStreamState = this
```

Comment:
When would this happen?

Reply:
I agree, it's a bit hand-wavy to just ignore those signals here.

IIRC, it could happen in cases, when a stream is cancelled before the response has been sent out. Right now this kind of coordination between incoming and outgoing stream side is hard to coordinate because the split of the two ends. In the commit comment I refer to the cleanup planned with #3232 to look into these kind of scenarios more deeply.